perllib/FixMyStreet/Cobrand/Merton.pm

```diff
@@ -62,21 +62,33 @@ sub reopening_disallowed {
 sub open311_extra_data_include {
     my ($self, $row, $h) = @_;
 
-    my $open311_only = [
-        { name => 'service', value => $row->service },
-    ];
-
     # Reports made via FMS.com or the app probably won't have a USRN
     # value because we don't access the USRN layer on those
     # frontends. Instead we'll look up the closest asset from the WFS
     # service at the point we're sending the report over Open311.
     if (!$row->get_extra_field_value('usrn')) {
         if (my $usrn = $self->lookup_site_code($row, 'usrn')) {
             $row->update_extra_field({ name => 'usrn', value => $usrn });
+            $row->update;
```

Comment:
I guess it doesn't matter, as we haven't done anything else at this point, but this line should not be necessary, see my other comment. Writing a test would be ideal, doesn't look like this function is tested at all.